### ISSUE TITLE
Error: Code generation from strings disallowed for this context

### ISSUE THREAD

**Issue by qeelei** [davideicardi/live-plugin-manager#83]
I am trying to install and use the `lodash` package:

```
    const manager = new PluginManager()
    await manager.install('lodash')

    const _ = manager.require('lodash')
    console.log('defaults: ', _.defaults({ a: 1 }, { a: 3, b: 2 }))
   
```

But the following error is thrown:

`Uncaught (in promise) Error: Code generation from strings disallowed for this context`

This `moment` example works fine

[https://github.com/davideicardi/live-plugin-manager#usage](url)

**Comment by davideicardi** [comment]
You are running both examples using the same runtime? What If you try to run the unit tests?

**Comment by qeelei** [comment]
@davideicardi  Yes it is in the same runtime. I am using the `Electron` framework, and these examples are running within a script called [preload](https://www.electronjs.org/docs/latest/tutorial/tutorial-preload).

The following is the result after I execute `npm run test`：
<img width="2116" alt="Screen Shot 2023-10-27 at 4 32 28 PM" src="https://github.com/davideicardi/live-plugin-manager/assets/3895769/1a00549c-ade7-4d6c-8308-7374c4dec1cb">

And when I only `install` the packages without `require`, the error mentioned in my question is not thrown.
```
    const manager = new PluginManager()
    await manager.install('lodash')

    // const _ = manager.require('lodash')
    // console.log('defaults: ', _.defaults({ a: 1 }, { a: 3, b: 2 }))
```
